lgc/patch/FragColorExport.cpp

```diff
@@ -699,7 +699,7 @@ Value *LowerFragColorExport::generateReturn(Function *fragEntryPoint, BuilderBas
   // color format in the metadata.
   m_pipelineState->getPalMetadata()->addColorExportInfo(m_info);
 
-  auto retInst = fragEntryPoint->back().getTerminator();
+  auto retInst = builder.GetInsertPoint()->getParent()->getTerminator();
```

Comment:
How about asserting that this is actually a ReturnInst?

Reply:
I've made that change in https://github.com/GPUOpen-Drivers/llpc/pull/980.